core/services/capreg/syncer.go

```diff
@@ -0,0 +1,94 @@
+package capreg
```

Comment:
What do you think about putting it in core/capabilities dir? There's a fake Syncer object already there, with some hardcoded values for our Staging tests - you could use them as an example/test.